handbook/people-ops/spending_company_money.md

```diff
@@ -0,0 +1,3 @@
+# Spending company money
```

Comment:
You can rename this doc, I just used/copied what GitLab used at https://about.gitlab.com/handbook/spending-company-money/.